account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -1,12 +1,12 @@
 import { custom, parseEther, publicActions } from "viem";
-
 import { LocalAccountSigner, type SmartAccountSigner } from "@aa-sdk/core";
-
-import { createSMAV2AccountClient } from "./client.js";
-
+import { createSMAV2AccountClient, type SMAV2AccountClient } from "./client.js";
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'SMAV2AccountClient' is defined but never used.
<!-- __reviewdog__:ChAyYjM2YjQwNDJhNDU1Mzk1EgZyZGpzb24= -->